### PR TITLE
ibmcloud-login: add page

### PR DESCRIPTION
**Pull request by Holomech**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/ibmcloud-login.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# ibmcloud login
+
+> Log in to the IBM Cloud.
+> More information: <https://cloud.ibm.com/docs/cli?topic=cli-ibmcloud_cli#ibmcloud_login>.
+
+- Log in by using interactive prompts:
```
Comment (marchersimon) [inline pages/common/ibmcloud-login.md:6]: Are there multiple prompts? If not I would use
```diff
- Log in by using interactive prompts:
+ Log in by using an interactive prompt:
```

File: pages/common/ibmcloud-login.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`ibmcloud login`
+
+- Log in to a specific API endpoint (default is `cloud.ibm.com`):
+
+`ibmcloud login -a {{api_endpoint}}`
+
+- Log in by providing username, password and targeted region as parameters:
```
Comment (marchersimon) [inline pages/common/ibmcloud-login.md:14]:
```suggestion
- Log in by providing username, password and the targeted region as parameters:
```

File: pages/common/ibmcloud-login.md (hunk 1 above)
Comment (Holomech) [reply pages/common/ibmcloud-login.md:6]: Well, there is a prompt for a username, followed by one for a password and then one for a region, so I thought that counts as multiple prompts.

File: pages/common/ibmcloud-login.md (hunk 2 above)
Comment (Holomech) [reply pages/common/ibmcloud-login.md:14]: Oops, missed that.

File: pages/common/ibmcloud-login.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`ibmcloud login -u {{username}} -p {{password}} -r {{us-south}}`
+
+- Log in with an API key, providing it as a string on the command line:
```
[the remark this answers is not in the document]
Comment (Holomech) [reply pages/common/ibmcloud-login.md:18]: Yeah, that sounds better.

File: pages/common/ibmcloud-login.md (hunk 1 above)
Comment (marchersimon) [reply pages/common/ibmcloud-login.md:6]: I'm not sure if this would still count as one prompt. Multiple prompts sounds pretty odd IMO.

Comment (Holomech) [reply pages/common/ibmcloud-login.md:6]: OK, I'll change it.

**Review by marchersimon — APPROVED** [review]
Thanks @Holomech

**Review by bl-ue — APPROVED** [review]
We should probably use "Login" instead of "Log in"...the former is used 144 times across all of the pages while "Log in" is only used 16 times...

**Comment by marchersimon** [comment]
Is "Login" even correct? <https://www.merriam-webster.com> doesn't find it and <https://www.dictionary.com/browse/login> says it should be used as "Log in".

**Comment by Holomech** [comment]
The way I understand it, "login" should be used as a noun, and "log in" should be used as a verb. At least that's what I found on the proper usage.

**Comment by marchersimon** [comment]
@bl-ue should I do a PR to standardize this in all pages? Or do you want to adress this in #5848?

**Review by nicokosi — APPROVED** [review]
👍

**Comment by marchersimon** [comment]
Ping @bl-ue

**Comment by bl-ue** [comment]
>@bl-ue should I do a PR to standardize this in all pages? Or do you want to adress this in #5848?

@marchersimon I'd say open a new PR.

**Pull request merged**